packages/client/src/pages/Game/core/entities/asteroid/index.ts

```diff
@@ -0,0 +1,41 @@
+import { GameObject } from '../game-object';
+import { AsteroidConfig } from './types';
+
+export class Asteroid extends GameObject {
+  private rotateAngle: number;
+  private rotateVector: -1 | 1;
+  private rotateSpeed: number;
+  public distance = 0;
+  private readonly moveAngle: number;
+
+  constructor(config: AsteroidConfig) {
+    super(config);
```

Comment:
```suggestion
    super({ ...config, isAnimated: false );
```
Вот так. Анимации и `currentAnimation` для астероида можно будет везде удалить